### ISSUE TITLE
S3 destination saved format as delta lake (delta tables)

### ISSUE THREAD

**Issue by mustafa-rmd** [airbytehq/airbyte#16322]
My current requirement is to have the following data pipeline:
PostgreSQL (Source)
Air byte
Minio - S3 storage (Destination)
Apache spark configure with (Minio and Delta lake formatting) since spark doesn’t support ACID transactions.

The goals to have air bye move data from PostgreSQL (Source) to Minio storage (Destination) saved in delta format. Spark then will come and read data from S3 expected to be with delta format.

My main issue with the output format for Air bye S3 connector. Currently is only supports 3 data types: CSV, Avro and JSON Lines (JSONL).

What is the recommend way to solve this problem?  since I think, many companies are trying to build this data pipeline.
Is there plan to have this feature released in upcoming releases?
Should we implement this feature? If so, is there a good documentation of how to start about it?
Or, is there another method of going about it? 

Thanks,

**Comment by natalyjazzviolin** [comment]
Hi @mustafa-rmd , could you please edit your request to follow our feature request template? This will ensure all details are understood clearly. Thank you!

## Tell us about the problem you're trying to solve
What are you trying to do, and why is it hard? A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

## Describe the solution you’d like
A clear and concise description of what you want to see happen, or the change you would like to see

## Describe the alternative you’ve considered or used
A clear and concise description of any alternative solutions or features you've considered or are using today.

## Additional context
Add any other context or screenshots about the feature request here.

## Are you willing to submit a PR?

Remove this with your answer :-)

**Comment by mustafa-rmd** [comment]
### Problem 
Deltalake (Delta table) format is an essential format for many pipeline architecture epically for ones that uses apache Spark in their  pipeline.

### Solution
I would like Delta format to be added along with apache avro, Json, etc.

Describe the alternative you’ve considered or used
Not alternatives

Additional context
![image](https://user-images.githubusercontent.com/112797628/192494136-896538ac-8047-4839-8284-00c2b345f160.png)


Are you willing to submit a PR?
Yes

**Comment by misteryeo** [comment]
@dennyglee Noted in your discussion that you're adding this to your roadmap. Just wanted to confirm that you're planning to contribute here?

**Comment by dennyglee** [comment]
@misteryeo Yes, we are planning to contribute here - it may or may not be me personally, but feel free to ping me on this until we figure this out :)

**Comment by wkargul** [comment]
Hey @dennyglee is there any update on that?

**Comment by seunggs** [comment]
@dennyglee @mustafa-rmd Any updates on this by any chance?

**Comment by herry13** [comment]
Hi @dennyglee @mustafa-rmd Any updates on this feature request? I am using Airbyte & DeltaLake in production. So I would love to see this destination connector to be available as soon as possible. I'm willing to give you some hands if needed.

**Comment by NatElkins** [comment]
Just want to chime in that I'm also interested in this!

**Comment by arorapankaj** [comment]
Do we have any update on this feature request ?

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 365 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
